runtimes/aleph-debian-11-python/loading.html

```diff
@@ -0,0 +1,346 @@
+<html lang="en">
```

Comment:
This is a pretty large file, would it make sense to load the CSS and SVG from a CDN ? (ex: IPFS)

Reply:
Yes, we can move it to an IPFS CDN, but for simplicity for the users, I think that is better to maintain all the imports inside the file, and allow our users to know better how it works and how to customize.